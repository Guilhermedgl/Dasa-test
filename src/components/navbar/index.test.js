import React from 'react';
import { create } from 'react-test-renderer';
import Navbar from './index';

describe('<Navbar />', () => {

  it('should match with snapshot', () => {
    const component = create(<Navbar />);
    expect(component.toJSON()).toMatchSnapshot();
  });
  
  it('should not use class', () => {
    const component = create(<Navbar />);
    expect(component.toJSON().props.class).toBeFalsy();
  });
});