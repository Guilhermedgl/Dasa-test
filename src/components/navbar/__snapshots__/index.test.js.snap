// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> should match with snapshot 1`] = `
<nav
  className="navbar"
>
  <h1>
    Pokemons
  </h1>
</nav>
`;
